CMakeLists.txt

```diff
@@ -1256,7 +1256,12 @@ elseif(UNIX)
 endif()
 
 # The mixxx executable
-add_executable(mixxx WIN32 src/main.cpp)
+if(QT7 AND CMAKE_VERSION VERSION_LESS "3.21")
```

Comment:
QT7?

Reply:
Ups, that is a debug leftover.

Reply:
I think it is the best to rebase this out.